routes/url-api-routes.js

```diff
@@ -5,7 +5,7 @@ module.exports = function(app) {
 
 // GET route to get session ID
   app.get("/api/urls/:pw", function(req, res) {
-    db.user.findAll({
+    db.Url.findAll({
```

Comment:
this was probably bad, but i named all the models with lowercase letters. so it would be db.url, not db.Url. same for the user and password models